src/components/ebay-menu/template.marko

```diff
@@ -2,7 +2,7 @@
     <ebay-button w-id="button" class=data.buttonClass variant="expand" size=data.size priority=data.priority aria-expanded=String(data.expanded) aria-haspopup="true" aria-label=data.a11yText>
         <span class="expand-btn__cell">
             <!-- Convoluted markup to satisfy both Skin and Marko -->
-            <div if(data.icon) w-preserve-body class="expand-btn__icon">
+            <div if(data.icon) w-preserve-body class="btn__icon">
                 <span if(data.iconTag) w-body=data.iconTag body-only-if(true)/>
```

Comment:
`body-only-if(true)`?

Reply:
Marko magic. Hopefully can clean this up after this lands https://github.com/marko-js/marko/pull/1094